packages/client-sdk-nodejs/src/internal/vector-index-data-client.ts

```diff
@@ -325,6 +326,85 @@ export class VectorIndexDataClient implements IVectorIndexDataClient {
     }
   }
 
+  private static buildFilterExpression(
```

Comment:
Would have been nice if this logic could be in one place, but the instantiating the proto classes are completely different between grpc nodejs and web.

Reply:
yep.